react/src/Navigation.tsx

```diff
@@ -253,6 +258,11 @@ export default function Navigation({ signout, darkMode, toggleDarkMode }: Naviga
                         <Tooltip title={"Logged in as " + currentUser.username} arrow>
                             <AccountCircleIcon fontSize="large" />
                         </Tooltip>
+                        <Tooltip title="Log out">
```

Comment:
```suggestion
                        <Tooltip title="Sign out">
```
Match how it was before (and sign in)